tasks/main.yml

```diff
@@ -27,7 +27,7 @@
 - name: Install Docker CE packages
   yum:
     name:
-      - docker-ce
+      - "docker-ce-{{ docker_version }}"
       - docker-ce-cli
```

Comment:
We probably need to make sure that the docker-ce and docker-ce-cli versions are in sync. That appears to be recommended by Docker's install guide.

Reply:
I have incorporated the version into the docker-ce-cli installation in my latest commit, and tested the updated version.